internal/formats/common/cyclonedxhelpers/external_references.go

```diff
@@ -79,6 +95,16 @@ func decodeExternalReferences(c *cyclonedx.Component, metadata interface{}) {
 		meta.Homepage = refURL(c, cyclonedx.ERTypeWebsite)
 	case *pkg.GemMetadata:
 		meta.Homepage = refURL(c, cyclonedx.ERTypeWebsite)
+	case *pkg.JavaMetadata:
+		var digests []syftFile.Digest
+		ref := findExternalRef(c, cyclonedx.ERTypeBuildMeta)
+		for _, hash := range *ref.Hashes {
+			digests = append(digests, syftFile.Digest{
+				Algorithm: fmt.Sprintf("%v", hash.Algorithm),
```

Comment:
I could be missing something, but could this be:
```suggestion
				Algorithm: hash.Algorithm,
```